src/_pytest/_code/source.py

```diff
@@ -269,46 +264,7 @@ def getsource(obj, **kwargs):
 
 
 def deindent(lines, offset=None):
```

Comment:
this is indeed the case, are we ok breaking this api?

Reply:
`dedent` is in `_pytest._code.source`, so I don't believe it will break any code. @RonnyPfannschmidt's concern was with the `Source` object, which is exposed by `testdir`, but here it is not the case.

@RonnyPfannschmidt please correct me if I'm wrong. 👍

Reply:
ah ok, I was also going to remove `offset=None` in `Source.deindent` as well, that's slightly closer to a break but still nobody should be calling it :laughing:

Reply:
i all for removing that api